podvm/Dockerfile.podvm_binaries.rhel

```diff
@@ -41,6 +41,6 @@ RUN cd cloud-api-adaptor/podvm && \
 RUN tar czvf /podvm-binaries.tar.gz -C /src/cloud-api-adaptor/podvm/files usr/ etc/
 RUN tar czvf /pause-bundle.tar.gz -C /src/cloud-api-adaptor/podvm/files pause_bundle/
 
-FROM registry.access.redhat.com/ubi8/ubi:8.7
```

Comment:
Can you also add a new commit for CentOS to use centos9 stream and codeready-builder ?

Reply:
lets do CentOS in separate i hit weird issue with pkgs namings